primefaces-integration-tests/src/test/java/org/primefaces/integrationtests/TomcatDeploymentAdapter.java

```diff
@@ -70,6 +71,13 @@ public void startup() throws Exception {
         con.disconnect();
 
         Thread.sleep(2000);
+
+        // make screenshots of failed tests
+        if ("true".equals(System.getenv("CI"))) {
+            // CI is running - see https://docs.github.com/en/actions/learn-github-actions/variables#default-environment-variables
+            ScreenshotOnFailureExtension.path4Screenshots = "/tmp/pf_it/";
```

Comment:
That´s the second part i currently use to achive our target.

Reply:
i think we should add a property here: https://github.com/primefaces/primefaces/tree/master/primefaces-selenium#configuration

and when parsing the properties, we can check for ${...} and resolve as environment var